### ISSUE TITLE
Add DAVx5 to the docs

### ISSUE THREAD

**Issue by MisterY** [jelmer/xandikos#160]
Perhaps the docs are using the old name, but DAVx5 has been tested with Xandikos:

https://www.davx5.com/tested-with/xandikos

Might be useful to add to the docs. Unless someone beats me to it, I'd try this as I'm going through the docs.

**Comment by jelmer** [comment]
Yeah, it's listed as DAVDroid still rather DAVx5 - including with the old URL.

A PR to fix that would be great. See https://github.com/xandikos/homepage/blob/master/content/pages/clients.rst
